### PR TITLE
Elm format would open a panel if elm-format fails

### PR DESCRIPTION
**Pull request by stoivo**

### CODE REVIEW

**Comment by Bernardoow** [comment]
@stoivo 

Can you upload screenshoot from this panel?

**Comment by stoivo** [comment]
One example
screen shot 2017-05-27 at 12 26 27

**Comment by Bernardoow** [comment]
@stoivo NICEEE ! I Liked this !!

File: elm_format.py [hunk 1]
```diff
@@ -31,6 +31,22 @@ def run(self, edit):
...
+			panel_view.run_command('erase_view')
+
+			# elm-format should have a --no-color option https://github.com/avh4/elm-format/issues/372
+			errors = re.sub('\x1b\[\d{1,2}m', '', errors.strip().decode())
+
+			panel_view.run_command('append', {'characters': errors})
+			panel_view.set_read_only(True)
+			self.view.window().run_command("show_panel", {"panel": "output.elm_format"})
+		else:
+			self.view.window().run_command("hide_panel", {"panel": "output.elm_format"})
```
Comment (sentience) [inline elm_format.py]: Looks great! 👍

Unfortunately, like the JSON files in this package, it seems this Python source file has some inconsistent indenting. Would you mind fixing that before we merge this?

Comment (stoivo) [inline elm_format.py]: Yes, it has. That's why I opened #26
Fixed

**Comment by stoivo** [comment]
Yes...

**Comment by stoivo** [comment]
Can we merge?

**Comment by sentience** [comment]
Thanks, @stoivo! ❤️

**Pull request merged**